test/integration/install.integration.ts

```diff
@@ -0,0 +1,230 @@
+import {ux} from '@oclif/core'
+import {CLIError} from '@oclif/core/lib/errors/index.js'
+import {expect} from 'chai'
+import {existsSync} from 'node:fs'
+import {rm} from 'node:fs/promises'
+import {tmpdir} from 'node:os'
+import {join} from 'node:path'
+import {SinonSandbox, createSandbox} from 'sinon'
+
+import PluginsIndex from '../../src/commands/plugins/index.js'
+import PluginsInstall from '../../src/commands/plugins/install.js'
+import PluginsUninstall from '../../src/commands/plugins/uninstall.js'
+
+describe('install/uninstall integration tests', () => {
+  let sandbox: SinonSandbox
+  let stubs: ReturnType<typeof ux.makeStubs>
+
+  const cacheDir = join(tmpdir(), 'plugin-plugins-tests', 'cache')
+  const configDir = join(tmpdir(), 'plugin-plugins-tests', 'config')
+  const dataDir = join(tmpdir(), 'plugin-plugins-tests', 'data')
+
+  before(async () => {
+    try {
+      await Promise.all([
+        rm(cacheDir, {force: true, recursive: true}),
+        rm(configDir, {force: true, recursive: true}),
+        rm(dataDir, {force: true, recursive: true}),
+      ])
+    } catch {}
+  })
+
+  beforeEach(() => {
+    sandbox = createSandbox()
+    stubs = ux.makeStubs(sandbox)
+    process.env.MYCLI_CACHE_DIR = cacheDir
+    process.env.MYCLI_CONFIG_DIR = configDir
+    process.env.MYCLI_DATA_DIR = dataDir
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+
+    delete process.env.MYCLI_CACHE_DIR
+    delete process.env.MYCLI_CONFIG_DIR
+    delete process.env.MYCLI_DATA_DIR
+  })
+
+  describe('basic', () => {
+    it('should return "No Plugins" if no plugins are installed', async () => {
+      await PluginsIndex.run([], process.cwd())
+      expect(stubs.stdout.firstCall.firstArg).to.equal('No plugins installed.\n')
```

Comment:
these can be brittle when something else hits stdout (warnings about updates, etc).

Maybe flatmap the callsArgs, and assert the presence of the message you expect in the array?